python/lsst/ts/scheduler/telemetry_stream_handler.py

```diff
@@ -0,0 +1,270 @@
+# This file is part of ts_scheduler
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Please refer to Vera C. Rubin Observatory. Check the other python files as well.